### DISCUSSION TITLE
[Android] How to output android Kotlin code debug log with `tauri android dev` ?

### DISCUSSION THREAD

**Discussion by tommyZZM** [tauri-apps/tauri#9449] [Q&A]
[Android] How to output android Kotlin code debug log with `tauri android dev` ?

**Comment by FabianLars** [comment]
I may misremember something but i think using the log plugin will help: https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/log (you only have to register it, no need to call it's actual logging apis)
